meta-client/src/main/java/org/apache/helix/metaclient/api/MetaClientInterface.java

```diff
@@ -515,5 +515,9 @@ default boolean subscribeOneTimeChildChanges(String key, ChildChangeListener lis
    */
   boolean waitUntilExists(String key, TimeUnit timeUnit, long timeOut);
 
+  public byte[] serialize(T data, String path);
```

Comment:
Neat : do you think API documentation can be added here just like other APIs in this interface?

Reply:
TFTR. Updated.